challenges/2022-11-08-common-elements/solution.test.js

```diff
@@ -0,0 +1,34 @@
+const { commonElements } = require('./solution');
+
```

Comment:
Test in case numbers repeated in both inputs should be repeated in the output:

```javascript
test('should handle repeated numbers', () => {
  const arr1 = [3, 3, 4, 6, 7, 9];
  const arr2 = [1, 2, 2, 2, 4, 10];
  const expected = [3, 3, 4];
  const actual = commonElements(arr1, arr2);
  expect(actual).toEqual(expected);
});
```

Reply:
@ericwburden I'm operating under the assumption that there should be no repeats. I'll add to the description to make this clear.